src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -1145,10 +1295,14 @@ def forward(self, hidden_states: torch.Tensor, input_tensor: torch.Tensor) -> to
 
 
 class LongformerAttention(nn.Module):
-    def __init__(self, config, layer_id=0):
+    def __init__(self, config, layer_id=0, cross_modality = False):
         super().__init__()
         self.self = LongformerSelfAttention(config, layer_id)
-        self.output = LongformerSelfOutput(config)
+        self.cross_modality = cross_modality
+        self.self_output = LongformerSelfOutput(config)
+        if cross_modality:
```

Comment:
Does `cross_modality` sound like a misnomer? I think it's just `cross_attention` considering both the modalities are text.